pyproject.toml

```diff
@@ -40,10 +40,11 @@ rouge-score = "^0.1.2"
 detoxify = "^0.5.2"
 langchain = "^0.3.7"
 asyncio = "^3.4.3"
-sentence-transformers = "^2.7.0"
+sentence-transformers = ">=3.10"
 typing = {version = "^3.7.4", python = "<3.10"}
 tiktoken = "^0.7.0"
 transformers = ">=4.52.1"
+torch = "2.7.1"
```

Comment:
This is motivated by the Snyk SCA scan, see https://security.snyk.io/vuln/SNYK-PYTHON-TORCH-9634167